spec/support/capybara_extensions.rb

```diff
@@ -70,15 +70,15 @@ def has_govuk_error_field?(model, field, error_text = nil)
     end
 
     def has_govuk_detail_summary?(text, options = {})
-      has_selector?(detail_summary_selector, **options.merge(text:))
+      has_selector?(detail_summary_selector, text:, **options)
```

Comment:
there is not need to merge text into options.
It's possible to pass it in as argument.